src/main/java/com/tabgok/heimdallr/HeimdallrServer.java

```diff
@@ -0,0 +1,20 @@
+package com.tabgok.heimdallr;
+
+import com.tabgok.entity.factory.MachineFactory;
+import com.tabgok.server.Gateway;
+
+/**
+ * Starts up an instance of the Heimdallr system, which consists of two parts:
```

Comment:
I took out the bit about how it starts up and added some information about what the HeimdallrServer is responsible for